GCC and previous Clang successfully compile const member functions that
modify non-mutable fields when they are members of a templated class
that is never instantiated. Recent Clang (v11 onwards) produces errors
in this case.

The problematic functions are never used, so we work around this by
omitting them.

diff --git a/applications/bed/bedextract/src/ExtractRows.cpp b/applications/bed/bedextract/src/ExtractRows.cpp
index c5f81b6a..c15644ca 100644
--- a/applications/bed/bedextract/src/ExtractRows.cpp
+++ b/applications/bed/bedextract/src/ExtractRows.cpp
@@ -77,7 +77,6 @@ namespace {
     }
 
     bool Empty() const { return empty_; }
-    void Clear() const { empty_ = true; }
 
   private:
     bool empty_;
diff --git a/interfaces/general-headers/utility/BitMonitor.hpp b/interfaces/general-headers/utility/BitMonitor.hpp
index f9fefdda..0eb91810 100644
--- a/interfaces/general-headers/utility/BitMonitor.hpp
+++ b/interfaces/general-headers/utility/BitMonitor.hpp
@@ -78,6 +78,7 @@ namespace Ext {
       _open = 0;
     }
 
+#if 0
     inline std::size_t next_set(std::size_t start) const {
       start += 1;
       std::size_t bin = start/BASE;
@@ -110,6 +111,7 @@ namespace Ext {
         _open = 0;
       return next_set(0);
     }
+#endif
 
     inline std::size_t next_unset(std::size_t start) {
       start += 1;
diff --git a/interfaces/general-headers/utility/CharPooledMemory.hpp b/interfaces/general-headers/utility/CharPooledMemory.hpp
index 267bb604..7d6793a2 100644
--- a/interfaces/general-headers/utility/CharPooledMemory.hpp
+++ b/interfaces/general-headers/utility/CharPooledMemory.hpp
@@ -203,6 +203,7 @@ namespace Ext {
         }
 
     private:
+#if 0
       char*
       find_open(char const* c)
         {
@@ -254,6 +255,7 @@ namespace Ext {
           } // while
           return nullptr;
         }
+#endif
 
       friend struct PooledCharMemory2<nelements>;
 
@@ -466,6 +468,7 @@ namespace Ext {
           return clean(c, sz);
         }
 
+#if 0
       char*
       clean(char const* c, std::size_t need)
         {
@@ -492,6 +495,7 @@ namespace Ext {
           _dirty.clear();
           return nullptr;
         }
+#endif
 
       friend struct PooledCharMemory<nelements>;
 
